manuscript/draft.Rmd

```diff
@@ -574,20 +574,20 @@ The most commonly used methods for forecasting future biodiversity, SDMs and
 macroecological models, both produced worse forecasts than time-series models
 and simple baselines. This weakness suggests that predictions about future
 biodiversity change should be viewed with skepticism unless the underlying
-models have been validated temporally, via hindcasting and comparison with
+models are temporally validated, via hindcasting and comparison with
```

Comment:
What about taking this a step further to "temporally validated using hindcasting...". I think the editors simplification makes this flow more smoothly.